editor/components/block-list/block.js

```diff
@@ -435,6 +436,13 @@ export class BlockListBlock extends Component {
 							/>,
 						] }
 					</BlockCrashBoundary>
+					{ ( showUI ) &&
+						<div className="editor-block-list__mobile-tools">
```

Comment:
Can we extract this to a `BlockMobileToolbar` component? Just thinking this `BlockListBlock` is already big enough :)

Reply:
I'll see what I can do, though stepping out for the day now. Thanks Riad.